### PR TITLE
docs: add vime linear_logp blog

### PR DESCRIPTION
**Pull request by inaniloquentee**
## Summary
- add English and Chinese vime + RL-Kernel linear_logp blog posts with SVG figures
- link the blog from the README/docs entry points and MkDocs navigation
- clarify the fused backward workspace/latency wording in the linear_logp operator doc

## Tests
- pre-commit run --all-files
- mkdocs build --strict -f mkdocs.yaml
- mypy --ignore-missing-imports rl_engine/
- py -3.13 -m pytest rl_engine/tests/test_dispatch.py -v
- PYTEST_DISABLE_PLUGIN_AUTOLOAD=1 py -3.13 -m pytest tests/test_attention_correctness.py -q -rs
- py -3.13 -m pytest tests/test_attention.py -v -k "not large and not gpu"
- py -3.13 -m pytest tests/test_kv_cache_attention.py -v -k "not large and not gpu"

## Summary by CodeRabbit

* **Documentation**
  * Updated the docs navigation and blog index to include a new “Blog” section with the latest English and Chinese posts.
  * Published an announcement blog post for RL-Kernel `linear_logp` integration for vime, with the README and docs landing page linking to it.
  * Refined the linear-logp operator guide to clarify how the forward avoids full-vocabulary logits materialization and how backward implementations affect memory and performance.

### CODE REVIEW

**Review by Flink-ddd — APPROVED** [review]
Excellent work!  LGTM. Thank you for work, this is our first blog, A good start. cc @KJLdefeated @a-kaa @maxiaosong1124 @EthanZero2Hero PTAL

**Review by KJLdefeated — APPROVED** [review]
LGTM!
